### DISCUSSION TITLE
I forgot both my password and my email, help!

### DISCUSSION THREAD

**Discussion by islam-ab** [community/community#56475] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

All I know is my account username, I do not know which email I used to create the account. I tried multiple emails but can not find it, my account (the one I'm trying to recover) username is "islamabm", I forgot the password and I don't remember the email I created the account on since I did that a long time ago.
The reason this happened is becasue I clread my cache and my whole browsing history, and previously I relied on auto-fill to log into my account. Help me please I need the account back.

**Comment by radar-blip** [comment, marked as answer]
You can ask [support](https://support.github.com/contact/account) for help with this.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @islam-ab! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by aliyahpatl** [reply]
Hello, I encountered the same problem. Do you happen to know any solution?
